utils/create_fixture.py

```diff
@@ -0,0 +1,271 @@
+    else:
+        logger.setLevel(logging.INFO)
+
+    logger.info("Start creating fixture file for {}".format(database))
+
+    time_init = datetime.now(timezone.utc)
+    time_str = time_init.strftime(STRFTIME)
+
+    db = pymysql.connect(host, user, password, database)
+    tables = get_tables(db)
+
+    fixtures = []
+    for table in tables:
+        logger.debug("Table {} start".format(table))
+        fixtures += get_fixture(db, time_str, table)
+        logger.debug("Table {} finished".format(table))
+    db.close()
+
+    file_name = database + "_fixture.json"
+    write_json(fixtures, file_name)
+
+    time_finished = datetime.now(timezone.utc)
+    duration = time_finished - time_init
+    logger.info("Fixture file created: {}".format(file_name))
+    logger.info("Total duration: {}".format(duration))
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("host",
+                        help="MariaDB/MySQL host")
+    parser.add_argument("database",
+                        help="MariaDB/MySQL database")
+    parser.add_argument("-u", "--user",
+                        default="root",
+                        help="MariaDB/MySQL user (by default = 'root')")
+    parser.add_argument("-p", "--password",
+                        default="",
+                        help="MariaDB/MySQL password (by default = '')")
+    parser.add_argument('-g', dest='debug_mode', action='store_true',
+                        default=False, help='Enable debug mode')
+    args = parser.parse_args()
+
+    return args
+
+
+def get_tables(db):
+    """Get a list of tables
+
+    :param db: MySQL connection
+    :return: table
+    """
+    tables = []
+    try:
+        logger.debug("MySQL: {}".format(QUERY_TABLES_SHOW))
+        cursor = db.cursor()
+        cursor.execute(QUERY_TABLES_SHOW)
+        data = cursor.fetchall()
+        for d in data:
+            tables.append(d[0])
+    except Exception as e:
+        logger.error("MySQL: {}: {}".format(QUERY_TABLES_SHOW, e))
+    finally:
+        logger.debug("Tables: {}".format(tables))
+        return tables
+
+
+def get_fixture(db, now, table):
+    """Create fixture given a table
+
+    :param db: MySQL connection
+    :param now: Datetime now
+    :param table: table
+    :return: fixture
+    """
+    fixture = []
+    model = get_model(table)
+    if not model:
+        return fixture
+
+    headers = get_headers(db, table)
+
+    try:
+        logger.debug("MySQL: {}".format(QUERY_SEARCH.format(table)))
+        cursor = db.cursor()
+        cursor.execute(QUERY_SEARCH.format(table))
+        data = cursor.fetchall()
+        count = 0
+        for d in data:
+            new = {
+                "fields": {},
+                "model": model
+            }
+            for index, h in enumerate(headers):
+                value = d[index]
+                if type(d[index]) == datetime:
+                    value = d[index].strftime(STRFTIME)
```

Comment:
I think data can be exported into a dictionary that for each row, contains the name of the columns and their values. Check [DictCursor](http://mysql-python.sourceforge.net/MySQLdb-1.2.2/public/MySQLdb.cursors.DictCursor-class.html)

Reply:
`fetchallDict(self): Fetch all available rows as a list of dictionaries. Deprecated: Use `fetchall()` instead. Will be removed in 1.3.`

Also, we have to change the value when the type is datetime